### ISSUE TITLE
cagent.exe crashing

### ISSUE THREAD

**Issue by clong** [mitre/caldera-agent#3]
The service starts "successfully", but I see the following error in Application event logs:

```
The description for Event ID 255 from source cagent cannot be found. Either the component that raises this event is not installed on your local computer or the installation is corrupted. You can install or repair the component on the local computer.

If the event originated on another computer, the display information had to be saved with the event.

The following information was included with the event: 

ERROR:root:Task was destroyed but it is pending!
task: <Task pending coro=<CalderaAgent.send_clients() done, defined at C:\Gitlab Runner\builds\f8e514af\0\caldera\caldera-agent\caldera_agent\agent.py:59> wait_for=<Future pending cb=[Task._wakeup()]>>

the message resource is present but the message is not found in the string/message table
```
Unfortunately, the agent never successfully connects to the server.

This was previously working, so I'm not sure what has changed that is causing this error. This is happening on Server2016 and Windows10.

**Comment by SpaceAether** [comment]
Hello, I am having the same issue as you.
How did you fix it ?

**Comment by dm-mitre** [comment]
The error message referenced above doesn't look like the base error. @clong I see you closed the issue in clong/DetectionLab#82 were you able to fix it?

**Comment by clong** [comment]
I guess I did, but to be honest I have no idea how it got fixed. I'll go ahead and close this issue - thanks for checking in!
